src/executor.pas

```diff
@@ -1871,6 +1871,9 @@ procedure TExecutor.InitSetOptions;
   FOptions.Insert(ANA_SO_BROWSER_OBS_DELETED_COLOR,      TFontColorOption.Create('#FF0000', rtString));
   FOptions.Insert(ANA_SO_BROWSER_OBS_VERIFIED_COLOR,     TFontColorOption.Create('#008080', rtString));
 
+  // Common chart options
+  FOptions.Insert(ANA_SO_CHART_COLORS,                   TSetOption.Create('123456789', rtInteger));
```

Comment:
Yes, but can we leave this for a future release? Or at least a future branch, so I can really check it out. Too much to change right now...

Reply:
I would prefer this is in before we make the next release, but I would also prefer this in a separate commit. This will simplify the review process too.

Reply:
I agree. For now, this set parameter is of type rtString.